src/styles/_catalog.scss

```diff
@@ -14,6 +14,13 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+
+    transition: transform 300ms, color 300ms;
```

Comment:
Would be nice to create a variable for duration, then we can use it for every transition to make it's duration consistent with other page transitions

Reply:
@OleksiiBlezniuk how can i open this pull request? i don't know how it closed. now I can't do 'npm run deploy'. help me ))

rootya@Rootya MINGW64 ~/projects/layout_catalog_hovers (develop)
$ git push origin develop 
To github.com:ArturNagaycev/layout_catalog_hovers.git
 ! [rejected]        develop -> develop (non-fast-forward)
error: failed to push some refs to 'github.com:ArturNagaycev/layout_catalog_hovers.git'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. Integrate the remote changes (e.g.
hint: 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.